torch/fx/passes/shape_prop.py

```diff
@@ -21,6 +21,9 @@ class TensorMetadata(NamedTuple):
     qscheme : Optional[torch.qscheme]
     q_scale : Optional[float]
     q_zero_point : Optional[int]
+    q_per_channel_scales : Optional[torch.Tensor]
```

Comment:
Hi @jamesr66a, it looks like TensorMeta needs to be backward compatible, how can we add new things to it?

Reply:
Add your class member with a default value that's assigned in the body of the class: https://docs.python.org/3/library/typing.html#typing.NamedTuple

After you do that, `--accept` test_class_member_back_compat and tag me for review